shared/chat/conversation/info-panel/participants.js

```diff
@@ -38,6 +38,7 @@ const Participants = ({participants, onShowProfile, onAddParticipant, style}: Pr
                 type="BodySemibold"
                 users={[{username, you: isYou, following, broken}]}
                 containerStyle={{marginLeft: 12}}
+                onUsernameClicked={onShowProfile}
```

Comment:
I guess on desktop the clickable property is "inherited" by the containing ClickableBox, but not on native.